static/app/views/issueList/utils/useFetchOrganizationFeatureFlags.tsx

```diff
@@ -0,0 +1,39 @@
+import {useFetchOrganizationTags} from 'sentry/actionCreators/tags';
+import type {Tag} from 'sentry/types/group';
+import type {UseApiQueryOptions} from 'sentry/utils/queryClient';
+import {Dataset} from 'sentry/views/alerts/rules/metric/types';
+
+export default function useFetchOrganizationFeatureFlags(
+  {
+    orgSlug,
+    projectIds,
+    enabled,
+    keepPreviousData = false,
+    useCache = true,
+    ...statsPeriodParams
```

Comment:
You only need to destructure the optional args

```suggestion
    keepPreviousData = false,
    useCache = true,
    ...params
```